docs/assets/FormValidator.js

```diff
@@ -46,16 +46,16 @@ FormValidator.prototype.showSummary = function() {
 }
```

Comment:
This is utterly misleading. This file has not been renamed; it's been deleted.

`docs/assets/FormValidator.js` is _generated_ from `assets/FormValidator.js`, using `npm run build`.